django/contrib/admin/views/main.py

```diff
@@ -123,10 +123,13 @@ def __init__(
             )
         self.to_field = to_field
         self.params = dict(request.GET.items())
+        self.filter_params = dict(request.GET.lists())
```

Comment:
`self.params` is now only use for ordering. Do you think it's worth keeping it? :thinking:
Maybe we could use this instead:
```python
self.order_params = request.GET[ORDER_VAR] if ORDER_VAR in request.GET else None
```

Reply:
It's also used in the `date_hierarchy` templatetag and a couple of `search_form.html` inserts.
I can still add `self.order_params` and try to migrate the stuff to the `filter_params` 

But I guess removing it could cause issues if anyone is using this in some overriding of inserts and stuff :thinking: what do you think?

Reply:
> But I guess removing it could cause issues if anyone is using this in some overriding of inserts and stuff thinking what do you think?

As far as I'm aware, it is not documented, so we can theoretically remove it :shrug: , but fine, let's leave it :+1: